handbook/engineering/incidents/index.md

```diff
@@ -19,11 +19,14 @@ Incidents can be reported by anyone (e.g. customers, Sourcegraph teammates) by a
 2. Start an internal communication thread about this incident in the #dev-ops channel in Slack.
     - All subsequent communication about this issue should happen in that Slack _thread_ (not in the top level #dev-ops channel).
 3. Identify an engineer to triage the incident.
-    - If you are an engineer and available for 30 minutes, then you should [triage the incident](#triage).
-    - If you are not an engineer or are not available to triage the incident, then ask the on-call engineer to triage the incident.
+    1. If the initial alert came from our alerting infrastructure, then the alert should be routed to the responsible [team](https://sourcegraph.com/search?q=repo:github.com/sourcegraph/sourcegraph+file:monitoring/frontend.go+ObservableOwner&patternType=literal)
+
+    1. If you are an engineer and available for 30 minutes, then you should [triage the incident](#triage).
+    1. If you are not an engineer or are not available to triage the incident, then ask the on-call engineer to triage the incident.
```

Comment:
This isn't really an ordered list, it is an enumeration of options, so I think unordered is better.

```suggestion
    - If the initial alert came from our alerting infrastructure, then the alert should be routed to the responsible [team](https://sourcegraph.com/search?q=repo:github.com/sourcegraph/sourcegraph+file:monitoring/frontend.go+ObservableOwner&patternType=literal)

    - If you are an engineer and available for 30 minutes, then you should [triage the incident](#triage).
    - If you are not an engineer or are not available to triage the incident, then ask the on-call engineer to triage the incident.
```